### ISSUE TITLE
When searching for a game there is an Error: SteamDB responed with a status of 403

### ISSUE THREAD

**Issue by mato200** [Racide/lumaManager#2]
I've read the forum thread and found no solutions to my problem

**Comment by Racide** [comment]
True, this little tool is currently in a non-working state (same cloudflare protection mentioned above).
Some preliminary work to migrate to Steam API has already been done, but now everything is on hold due to classes/impending exams..

**Comment by mato200** [comment]
btw, this being "yet another lumaManager ". Are there any other ones i could use in the meanwhile ?

**Comment by Racide** [comment]
Yup, this tool is inspired (_and compatible with, if you'll ever decide to switch back_) by ImaniiTy's [GreenLuma-Reborn-Manager](https://github.com/ImaniiTy/GreenLuma-Reborn-Manager). There might be others on github.

I remember It not working reliabily for me, things may have changed in the meanwhile.

This one was supposed to be easily tweakable, but the activation of CloudFlare's checks kinda made it pointless.
But no biggie, switching to the proper APIs will make for a better tool, with a small cost in code complexity and user experience.
